sdks/python/apache_beam/runners/portability/sdk_container_builder.py

```diff
@@ -236,10 +235,21 @@ def invoke_docker_build_and_push(self, container_image_name):
     build.timeout = Duration().FromSeconds(seconds=1800)
 
     now = time.time()
-    _LOGGER.info('Building sdk container, this may take a few minutes...')
+    _LOGGER.info(
+        'Building sdk container with google cloud build, this may '
+        'take a few minutes...')
     operation = client.create_build(project_id=project_id, build=build)
     # if build fails exception will be raised and stops the job submission.
-    result = operation.result()
+    try:
+      result = operation.result()
+    except Exception as e:
+      build_lists = client.list_builds(
```

Comment:
Why list_ _builds vs get_build using the operation build ID here?

Reply:
get_build requires a build id that is not returned from calling client.create_build(), this may be a usability issue from the python client but I didn't find any thing simpler